src/background/responders.ts

```diff
@@ -119,6 +164,10 @@ export const responders: { [T in Action['type']]: Responder<T> } = {
         feedbackState: {
```

Comment:
Let's make a function `emptyFeedbackState` that returns this whole object, which we now see repeated several times.